requirements.txt

```diff
@@ -1,38 +1,55 @@
 # For outdated module report see https://requires.io/github/OpenTreeMap/otm-core/requirements/?branch=develop
 # Changelog URL shown below if not linked on above page
 
-boto==2.42.0                             # http://docs.pythonboto.org/en/latest/releasenotes/v2.39.0.html
-# https://docs.djangoproject.com/en/1.10/releases/#id2
-Django==1.11.3  # rq.filter: >=1.11,<1.12
-django-apptemplates==1.2
+amqp==1.4.9
+anyjson==0.3.3
+billiard==3.3.0.23
```

Comment:
This should be fixed by https://github.com/OpenTreeMap/otm-core/pull/3184/commits/1147a7e3a8ad5d4f60a7f4b5e5c80857795f7df2